### PR TITLE
docs: A variety of docs updates (various nits)

### LINKED ISSUE #2932: Variety of content updates for Gaia docs
# Problem

A variety of minor issues were found in Gaia docs during the Docusaurus migration, these are detailed below.

# Closing criteria

All issues below are fixed.

# Problem details
- Missing footer logo on main page
- Remove outdated/defunct wallets and explorers from the main section
- [/governance/proposal-types](https://msalopek.github.io/gaia/governance/proposal-types) is incorrectly named by “Parameter changes” in the Governance menu and the links for Software Upgrade IBC Client Update proposal types are missing.
- [/#cosmos-hub-wallets](https://msalopek.github.io/gaia/#cosmos-hub-wallets)
  - Keplr and Ledger have Android, iOS apps
  - “Metamask Snaps” - perhaps add a one-line explainer below
- [/getting-started/what-is-gaia](https://msalopek.github.io/gaia/getting-started/what-is-gaia): is this normal that the interchain-security and global-fee modules aren’t mentioned?
- [/hub-tutorials/gaiad#key-types](https://msalopek.github.io/gaia/hub-tutorials/gaiad#key-types): The “cosmospub” and “cosmosvalconspub” prefix are outdated, as they can’t be parsed using gaiad keys parse.
- [Migrations instructions ](https://msalopek.github.io/gaia/migration)page is a README and the instructions list isn’t sorted
- [/getting-started/installation#install-the-binaries](https://msalopek.github.io/gaia/getting-started/installation#install-the-binaries):  output are still for version v9.0.0
- [/hub-tutorials/join-testnet#version-history](https://msalopek.github.io/gaia/hub-tutorials/join-testnet#version-history) : table isn’t updated
- [/architecture/adr/adr-001-interchain-accounts](https://msalopek.github.io/gaia/architecture/adr/adr-001-interchain-accounts): the title is duplicated
- [/interchain-security](https://msalopek.github.io/gaia/interchain-security): add a link to the ICS https://interchainsecurity.dev/ page or gh repo

### PR DESCRIPTION
**Pull request by mmulji-ic**
## Description

Closes: #2932

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

File: docs/docs/architecture/adr/adr-001-interchain-accounts.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <!--
 order: 2
+title: ADR 001: Interchain Accounts
 -->
 
-# ADR 001: Interchain Accounts
-
```
Comment (mpoke) [inline docs/docs/architecture/adr/adr-001-interchain-accounts.md:6]: Why remove the title? When checking the page on GH, there will be no title.

File: docs/docs/roadmap/previous-releases.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 <!--
 order: false
+title: Previous Releases
 parent:
   title: Previous Releases
   order: 7
 -->
 
-# Previous Releases
-
```
Comment (mpoke) [inline docs/docs/roadmap/previous-releases.md:9]: ditto: The title is not visible on GH.

File: docs/docs/migration/README.md [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 parent:
-  title: Migration Instructions
   order: 9
 ---
 <!--
 markdown-link-check-disable
 -->
 
-# Migration Instructions
-
```
Comment (mpoke) [inline docs/docs/migration/README.md:12]: ditto: title on GH

File: docs/docs/interchain-security/README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ parent:
   order: 2
 ---
 
-The Interchain Security feature, brings to the Cosmos Hub a shared security model, where the Cosmos Hub validators, also validate on consumer chains. This is valuable for consumer chains, as consumer chains can focus on product-market fit, rather than business and operational agreements in bringing together a validator set. As part of this agreement, consumer chains pay for the security by distributing a portion of the consumer chain revenue to Hub token holders.
+The [Interchain Security](https://interchainsecurity.dev/) feature, brings to the Cosmos Hub a shared security model, where the Cosmos Hub validators, also validate on consumer chains. This is valuable for consumer chains, as consumer chains can focus on product-market fit, rather than business and operational agreements in bringing together a validator set. As part of this agreement, consumer chains pay for the security by distributing a portion of the consumer chain revenue to Hub token holders.
```
Comment (mpoke) [inline docs/docs/interchain-security/README.md]: Link to ICS doc page instead, at least until we fix the ICS webpage.

File: docs/docs/roadmap/previous-releases.md (hunk 2 above)
Comment (mmulji-ic) [reply docs/docs/roadmap/previous-releases.md:9]: To keep a coherent title in Docusaurus menu on the left hand side, we use the title in markdown metadata, if we add it for GH, the title is duplicated as shown here.
image

**Review by p-offtermatt — APPROVED** [review]

**Pull request closed without merging**